libtest/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(LIBTEST_HEADERS
     queue_utils_lib.h
     stopwatch.h
     cr_template.h
+    grab-logging.h
```

Comment:
[optional] Only if you modify the patches anyway, and you feel like it:  this would be better in the previous patch (where you extracted the internal message logging).